content/en/docs/create-project/_index.md

```diff
@@ -1,9 +1,38 @@
 ---
 title: "Creating projects"
-date: 2017-01-05
 weight: 30
 description: >
-  Now that you've setup the platform, let's create your first project.
+  Now that you've setup the platform, let's work with your projects.
 aliases:
   - /docs/getting-started/first-project/
 ---
+
+Whether you're new to development or have multiple existing projects, Jenkins X can help you automate your building, testing, and deployments quickly and easily!
+
+### Quickstarts
+
+If you're just starting your development process with Jenkins X and want to use some tested, supported project templates that you can setup quickly, you can create a **Quickstart**!
+
+Quickstarts are pre-made applications you can create a project from, instead of making one from scratch.
+
+{{< cards >}}
+  {{< card-body card-title="Node HTTP" card-detail="The HTTP interfaces in Node.js are designed to support many features of the protocol which have been traditionally difficult to use." url="/docs/create-project/creating/" fa-symbol="cube" >}}
+
+  {{< card-body card-title="Spring Boot" card-detail="Spring Boot is Spring's convention-over-configuration solution for creating stand-alone, production-grade Spring Applications." url="/docs/create-project/creating/create-spring" fa-symbol="leaf" >}}
```

Comment:
I think we can actually delete the create-spring page. The reasons are:

- `jx create spring` as documented on this page is deprecated and one should use `jx create quickstart` selecting the appropriate Spring quickstart. There is no reason why it would make sense to treat Spring in any ways special here. It is also more consistent
- If one removes the deprecated `jx create spring` commands from the _create-spring_ page, one is left with "Importing existing Spring Boot projects" which is no different then a generic import.